docs/documentation/saros-server.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Saros-Server
+---
+
+**Note**: This is an *alpha* feature and not intended for general use yet. However please feel free to play around and give some initial feedback. Contributers to the Saros-Server project are very welcome and might want to take a look at [contribution wiki](../contribute/saros-server.md) for further information.
+
+## Use-case
+
+The use-case of the server is to host session independently of any participant. As it takes the role of the session-host any participant may leave or join the session at any time.
+
+## Usage
+
+The Saros-Server is currently available in version **0.1.0** as a standalone jar on [our Github Releases](https://github.com/saros-project/saros/releases).
+
+The server needs it's own Jabber-ID and can then be started via:
```

Comment:
Even if we call it a JID( = Jabber ID) I would prefer to call it an "XMPP account" as in the getting started docs.
```suggestion
The server needs it's own XMPP account and can then be started via:
```